.rubocop_todo.yml

```diff
@@ -136,6 +136,8 @@ Lint/UselessAssignment:
 # Configuration parameters: AllowedMethods, AllowedPatterns, CountRepeatedAttributes.
 Metrics/AbcSize:
   Max: 143
+  Exclude:
+    - 'app/serializers/initial_state_serializer.rb'
```

Comment:
I think this was manually added, so it will be wiped out the next time someone regenerates the file.

Reply:
I'm regenerating it now.

Reply:
The todo file I'm generating does not exclude the initial_state_serializer. Instead it increasse the AbcSize to 155.

Is that ok?